src/chainparams.cpp

```diff
@@ -84,7 +84,7 @@ class CMainParams : public CChainParams {
         consensus.nPowTargetSpacing = 90;
         consensus.fPowAllowMinDifficultyBlocks = false;
         consensus.fPowNoRetargeting = false;
-        consensus.nRuleChangeActivationThreshold = 13440; // 95%
+        consensus.nRuleChangeActivationThreshold = 13440; // 100% TODO: Is this good?
```

Comment:
For 2.0.x, please only set the comment to 100%
Please delete [ToDo: Is this good?]

Reply:
i removed the TODO message.